cpp/ql/test/query-tests/Security/CWE/CWE-079/semmle/CgiXss/search.c

```diff
@@ -37,14 +37,26 @@ void good_server(char* query) {
   puts(do_search(query));
 }
 
+int scanf(const char *, ...);
```

Comment:
```suggestion
int sscanf(const char *s, const char *format, ...);
```

Reply:
Jeez, I need to read what I actually write before I commit it >.< Thanks a lot!

Reply:
You were 'saved' by C implicit declarations!